docs/tutorials/substrate-front-end/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Introduction"
+id: substrate-front-end-index
```

Comment:
Is this necessary to make the sidebars come back?

Reply:
By default the page has id `introduction`. But ID should be unique across all doc pages, and `introduction` as a doc page ID just seem too generic to me. So I specify one.

Reply:
Great. We should be careful of that in the future. Glad you caught it.